STEER/Utilities/alihadd.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+# source $ALICE_ROOT/libexec/alihadd.sh
+# Example usage:
+#   source $ALICE_ROOT/libexec/alihadd.sh
+#    * Filter input list before merging
+#       makeFilteredList input.list highPt filter.log
+#    *  Internal function
+#       testFileList input.list highPt | tee  filter.log
+
+
+loadUtilities(){
+    #
+    #  load utilities if not yet done
+    if [ -z `type -t alilog_info` ]; then
+        source $ALICE_ROOT/libexec/alilog4bash.sh
+        alilog_info "Load ALICE_ROOT/libexec/alilog4bash.sh"
+    fi;
+    if [  -z `type -t guessPeriod` ]; then
+        source $ALICE_ROOT/libexec/utilities.sh
+        alilog_info "Load $ALICE_ROOT/libexec/utilities.sh"
+    fi;
+}
+
+testFileList(){
+    # Internal function: filter input list
+    # Provide good/bad file list out of inputList(param[$1])
+    # Tests: read/write/delete/eventually copytree for selected keys(param[$2])
+    inputList=$1
+    keyList=$2
+    timeOut=$3
+    alilog_info "inputList=$inputList"
+    alilog_info "keyList=$keyList"
+    for a in `cat $inputList`; do
+    alilog_info "Filter.Start $a $keyList $timeOut"
+    prefix=""
+    if [ $timeOut > 0 ] ; then
+        prefix="timeout $timeOut"
+    fi;
+    export ROOT_HIST=0;
+    $prefix aliroot -n -b   -l   << EOF
```

Comment:
Not sure I understood your comment.
I had to use timeout because sometimes file tests ends were stuck.
I was debugging what happenned - some locks sets - within ALICE error handler
(I can find in JIRA ticket) in case of exceptions. 
timeout was the only option which I found